package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@cowprotocol/app-data": "^0.0.1-RC.2",
     "@cowprotocol/contracts": "^1.3.1",
-    "cross-fetch": "^3.1.5",
+    "cross-fetch": "3.0.4",
```

Comment:
Why the downgrade?

Reply:
try it with that version it's giving `bind` is not found, on all tests using cross-fetch. this version fixes that